examples/platforms/da15000/README.md

```diff
@@ -15,29 +15,30 @@ $ make -f examples/Makefile-da15000
 
 Execute script to prepare image for flashing:
 
-```
+```bash
 $ cd ./third_party/dialog/DialogTools
 $ ./imgprep.sh
 ```
 
 Flash Binaries:
 
 1. Open another terminal window in order to start gdb server. Write the command (Linux):
-```
+```bash
 $ JLinkGDBServer -ir -if swd -speed auto -device Cortex-M0 -logtofile on -localhostonly
 ```
-
 2. Return to previous terminal window and execute a command in order to flash the board:
```

Comment:
This is still rendering as a second "1." bullet point due to there being unindented paragraphs between the list items.   Everything between the bullet points needs to be indented.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists